uint8 setpoint_index # 0=cargo ship, 1=low rocket, 2=mid rocket, 3=high rocket, 4=intake
uint8 end_setpoint_index  
---
bool timed_out
bool success
---
